### DISCUSSION TITLE
How can I use parsec benchmark?

### DISCUSSION THREAD

**Discussion by gihyeonjeon** [gem5/gem5#867] [Q&A] [answered]
Hi!
I'm proceeding with the parsec tutorial provided by gem5, but it's not working properly. I have two questions: the first is about the parsec directory. If I only refer to the parsec tutorial page, I didn't create a directory named parsec, yet the change directory command is used.
The second issue arises when I enter './packer build parsec/parsec.json'. There's an error because the page http://parsec.cs.princeton.edu/download/3.0/parsec-3.0-core.tar.gz cannot be found, and even when I enter this address into the actual address bar, the webpage does not appear. I'm not sure which address to change it to or which file needs to be modified.
I would like to know if there's a specific part I should look at to solve this problem.
Of course, I'm aware that the memory space created in this way is less efficient, but that is intentional.
Thanks! :D

**Comment by BobbyRBruce** [comment, marked as answer]
We already provide a pre-built Parsec Disk Image as part of gem5 Resources (for X86 at least): https://resources.gem5.org/resources/x86-parsec?version=1.0.0

Here is an example gem5 configuration script using this PARSEC benchmark resource: https://github.com/gem5/gem5/blob/stable/configs/example/gem5_library/x86-parsec-benchmarks.py

**Reply by gihyeonjeon** [reply]
Really thanks!..
Does it mean that I can download the PARSEC disk image from the provided link, and then use it to write and utilize a PARSEC run script?
Is my understanding correct?

**Reply by BobbyRBruce** [reply]
You could do that, yes.

**Reply by gihyeonjeon** [reply]
Really thanks!
